pixeltable/globals.py

```diff
@@ -365,6 +365,10 @@ def drop_dir(path_str: str, force: bool = False, ignore_errors: bool = False) ->
             raise e
 
     children = cat.paths.get_children(path, child_type=None, recursive=True)
+
+    if len(children) > 0 and not force:
+        raise excs.Error(f'Directory `{path_str}` is not empty.')
+
     if force:
```

Comment:
that check is now superfluous (ie, you can remove line 72 and unindent the block)